vmdb/db/migrate/20150501193927_default_provider_verify_ssl.rb

```diff
@@ -0,0 +1,9 @@
+class DefaultProviderVerifySsl < ActiveRecord::Migration
+  def up
+    change_column :providers, :verify_ssl, :integer, :null => false
```

Comment:
Yeah, please remove this migration as default_value_for handles this case.  We generally avoid schema-level contraints that may not be undoable in a release.